### PR TITLE
Implement switcher on home page & new impact group cards (Kids-802 & kids-801)

### PR DESCRIPTION
**Pull request by Daniela510**
## Description

Ready for review

### CODE REVIEW

File: lib/features/give/pages/home_page.dart [hunk 1]
```diff
@@ -100,7 +102,14 @@ class _HomePageState extends State<HomePage> {
     return Scaffold(
       key: _key,
       appBar: AppBar(
-        title: Text(isGive ? locals.amount : locals.discoverHomeDiscoverTitle),
+        title: switch (pageIndex) {
+          0 => Text(locals.amount),
+          1 => !auth.user.isUsUser
+              ? Text(locals.discoverHomeDiscoverTitle)
+              : Text('Choose group'),
```
Comment (pavelpavelko) [inline lib/features/give/pages/home_page.dart:109]: shall the text be in POEditor s well?

File: lib/features/give/widgets/animated_switch.dart [hunk 2]
```diff
@@ -20,41 +20,45 @@ class AnimatedSwitch extends StatelessWidget {
...
       height: 40,
-      indicatorSize: const Size(100, double.infinity),
-      borderRadius: const BorderRadius.all(Radius.circular(10)),
-      innerColor: AppTheme.givtGraycece,
-      borderColor: AppTheme.givtGraycece,
-      indicatorColor: AppTheme.givtLightGreen,
+      dif: 1,
+      indicatorSize: const Size.fromWidth(90),
+      borderWidth: 1,
+      borderColor: const Color(0xFFDCDCE1),
```
Comment (pavelpavelko) [inline lib/features/give/widgets/animated_switch.dart:27]: shall we move color to the AppTheme?

File: lib/features/give/widgets/home_goal_tracker.dart [hunk 3]
```diff
@@ -115,61 +75,61 @@ class HomeGoalTracker extends StatelessWidget {
...
+        Center(
+          child: Padding(
+            padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 32),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  group.organisation.organisationName ??
+                      'Oops, did not get a name for the goal.',
```
Comment (pavelpavelko) [inline lib/features/give/widgets/home_goal_tracker.dart:104]: POEditor?

File: lib/features/give/widgets/home_goal_tracker.dart [hunk 4]
```diff
@@ -115,61 +75,61 @@ class HomeGoalTracker extends StatelessWidget {
...
+                              fontWeight: FontWeight.w800,
+                              fontSize: 17,
+                            ),
+                  ),
                 ),
-              ),
+                const SizedBox(height: 4),
+                Text(
+                  group.type == ImpactGroupType.family
+                      ? 'Family Goal: \$${currentGoal.goalAmount}'
```
Comment (pavelpavelko) [inline lib/features/give/widgets/home_goal_tracker.dart:116]: POEditor?

File: lib/features/give/widgets/triple_animated_switch.dart [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      borderColor: const Color(0xFFDCDCE1),
+      borderRadius: const BorderRadius.all(Radius.circular(4)),
+      animationCurve: Curves.easeInOut,
+      colorBuilder: (i) => i == pageIndex ? AppTheme.givtBlue : Colors.white,
+      onChanged: onChanged,
+    );
+  }
+
+  Widget iconBuilder(int value, Size size) => switch (value) {
+        0 => _buildToggleSwitch('Give', 0),
```
Comment (pavelpavelko) [inline lib/features/give/widgets/triple_animated_switch.dart:37]: POEditor

File: lib/features/impact_groups/models/impact_group.dart [hunk 6]
```diff
@@ -36,13 +39,14 @@ class ImpactGroup extends Equatable {
       type: ImpactGroupType.fromString(map['type'] as String),
       description: map['description'] as String? ?? '',
       image: map['image'] as String? ?? '',
-      amountOfMembers: (map['amountOfMembers'] as num? ?? 0).toInt(),
-      organiser: map['organiser'] != null
-          ? GroupOrganiser.fromMap(map['organiser'] as Map<String, dynamic>)
+      amountOfMembers: (map['numberOfMemebers'] as num? ?? 0).toInt(),
+      organiser: map['organizer'] != null
```
Comment (pavelpavelko) [inline lib/features/impact_groups/models/impact_group.dart:43]: still organizer?

File: lib/features/impact_groups/pages/impact_group_screen.dart [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          SvgPicture.asset('assets/images/family_superheroes.svg'),
+          const SizedBox(height: 20),
+          Text(
+            'Your Family Group and other\ngroups will appear here',
```
Comment (pavelpavelko) [inline lib/features/impact_groups/pages/impact_group_screen.dart:68]: POEditor?

File: lib/features/impact_groups/widgets/impact_group_details_bottom_panel.dart [hunk 8]
```diff
@@ -17,51 +18,60 @@ class ImpactGroupDetailsBottomPanel extends StatelessWidget {
...
-                  ),
+      child: SafeArea(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Text(
+              context.l10n.goal,
+              //TODO: update when goal will have name fiald
+              // 'Goal${impactGroup.isFamilyGroup ? ': ${impactGroup.goal.orgName}' : ''}',
```
Comment (pavelpavelko) [inline lib/features/impact_groups/widgets/impact_group_details_bottom_panel.dart:31]: shall we uncomment it if we already have goal name to show?

File: lib/features/impact_groups/widgets/impact_group_details_bottom_panel.dart [hunk 9]
```diff
@@ -17,51 +18,60 @@ class ImpactGroupDetailsBottomPanel extends StatelessWidget {
...
+            const SizedBox(height: 15),
+            CustomGreenElevatedButton(
+              title: context.l10n.give,
+              onPressed: () {
+                AnalyticsHelper.logEvent(
+                  eventName: AmplitudeEvents.impactGroupDetailsGiveClicked,
+                  eventProperties: {'name': impactGroup.name},
+                );
+
+                //TODO: imtegrate with giving flow when navigation is ready
```
Comment (pavelpavelko) [inline lib/features/impact_groups/widgets/impact_group_details_bottom_panel.dart:56]: do you plan to work on giving flow in scope of other PR or?

File: lib/shared/models/user_ext.dart [hunk 10]
```diff
@@ -70,6 +70,8 @@ class UserExt extends Equatable {
         notificationId = '',
         profilePicture = '';
 
+  bool get isUsUser => Country.fromCode(country).isUS;
```
Comment (pavelpavelko) [inline lib/shared/models/user_ext.dart:73]: nice)

File: lib/features/impact_groups/models/impact_group.dart (hunk 6 above)
Comment (Daniela510) [reply lib/features/impact_groups/models/impact_group.dart:43]: nope

**Pull request closed without merging**